app/models/allowed_user_email_masks.rb

```diff
@@ -0,0 +1,5 @@
+class AllowedUserEmailMasks < ActiveRecord::Base
```

Comment:
What exactly is a mask?

Reply:
It's a regular expression that can be used to isolate/approve a string.  I'm not using regex, just the simple form of "@nypl.org is allowed now".  I wasn't sure the best way to name it -- "domain" seems not accurate...

Reply:
Maybe add a comment re-iterating that? Thoughts?

Reply:
Fair.

Reply:
done